android/gradle.properties

```diff
@@ -45,3 +45,5 @@ MYAPP_RELEASE_KEY_PASSWORD=sussolpikelets
 
 android.useAndroidX=true
 android.enableJetifier=true
+
+org.gradle.jvmargs=-XX\:MaxHeapSize\=1024m -Xmx1024m
```

Comment:
Related reference: https://scans.gradle.com/. Seems like a handy tool for debugging build issues.

Reply:
some developers who have a lot of cores need a larger heap size. Sorry if you don't need this?

Reply:
Nah, wasn't complaining about it, was just saying that this tool seems useful for debugging and possibly finding a good heap size.